app/views/businesses/show.html.erb

```diff
@@ -9,17 +9,38 @@
   </div>
 </div>
 
-<div class="container">
-  <div class="row">
-    <div class="col-sm-12">
-      <!-- Masonry Container -->
-      <div id="masonry-container">
-        <% 15.times do %>
-          <div class="box">
-            <%= link_to( image_tag("http://placehold.it/#{@items.sample}text=Item"), "#") %>
+<div class="row">
+    <% @items.each do |item| %>
+      <div class="col-sm-3 text-center item">
+        <div class = "divider">
+        
+        <% if current_user.nil? || !current_user.admin?%>
+          <% @item_show = item_path(item)  %>
+        <% else %>
+          <% @item_show = admin_item_path(item) %>
+        <% end %>
+
+        <h4><%= link_to item.title, @item_show %></h4>
+        <%= link_to image_tag(item.image, size: "200"), @item_show %>
+        <div class="row">
+          <p class = "spacer col-md-12"><em><%=item.description %></em></p>
+          <h4 class="text center"> $<%= item.price_in_dollars %> </h4>
+        </div>
+        <% if current_user.nil? || !current_user.admin?%>
```

Comment:
It doesn't matter since I'm re-writing the authorization now, but in the future I would use something like `current_user.default?` instead of `!current_user.admin?`. Just makes for cleaner and more understandable code.